deploy/operator/gather.sh

```diff
@@ -129,15 +129,15 @@ function gather_capi_data() {
     done
   done
 
-  oc logs deployment/operator -n hypershift > "${capi_dir}/hypershift.log" || true
+  oc logs --tail=-1 deployment/operator -n hypershift > "${capi_dir}/hypershift.log" || true
```

Comment:
are you sure it currently truncates?
so far we've only seen the need of this flag when using ``--selector``

Reply:
It is only to be on the safe side. The current code does not cause any problems. But also to be honest, the test where it runs is a fairly simple one so we never expect there to be a flood of logs. In the end feel free to push it or hold as you wish. From my perspective (as someone who just needs access to CAPI logs) it works as it is currently